.github/workflows/push-master.yml

```diff
@@ -0,0 +1,48 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (c) 2023 - 2023 by the IBAMR developers
+## All rights reserved.
+##
+## This file is part of IBAMR.
+##
+## IBAMR is free software and is distributed under the 3-clause BSD
+## license. The full text of the license can be found in the file
+## COPYRIGHT at the top level directory of IBAMR.
+##
+## ---------------------------------------------------------------------
+
+name: push
+on:
+  # things to do when we push to master
+  push:
+    branches:
+      - documentation-test-branch
```

Comment:
There's no good way to test this without first merging this branch so that GitHub actions start to run. For now, we'll run these steps (which might fail - I don't know what the state of the Travis instructions were) whenever this (non-master) branch is updated and once things are set up we'll switch this to 'master'.